Contributing.md

```diff
@@ -2,38 +2,32 @@
 ---
 ## Want to help? We would love that!
 Here are some ways to get involved, ranging in
-difficulty from easiest to hardest.
+difficulty from easiest to hardest
 
 ## Bug Hunting
 Run the software and tell us when it breaks. We are happy to receive bug
-reports. This software was developed for internal use
-on the fly as needed. This means that the code was not built to the
-typical standards of an open source project, but we would like to see it get
-there.
+reports
 
 Just be sure to do the following:
 * Check if the bug is already accounted for on the
 [Github issue tracker](https://github.com/activecm/rita/issues)
-  * If an issue already exists, add the following info in a comment
-  * If not, create an issue, and include the following info
+  * If an issue already exists, add the relevant info in a comment
+  * If not, create an issue and include the relevant info
 * Give very specific descriptions of how to reproduce the bug
   * Log files can be found at ~/.rita/logs
 * Include the output of `rita --version`
-* Include a description of your hardware (ex. CPU, RAM, filesystems)
-* Tell us about the size of the test, and the physical resources available
+* Include a description of your hardware (e.g. CPU, RAM, filesystems)
+* Tell us about the size of the test and the physical resources available
 
 ## Contributing Code
 There are several ways to contribute code to the RITA project.
 Before diving in, follow the [Manual Installation Instructions](docs/Manual%20Installation.md)
 
-* Add godoc comments and fix style compliance issues:
-  * Run the [go metalinter](https://github.com/alecthomas/gometalinter)
-  * Find a linting error and fix it
 * Work on bug fixes:
   * Find an issue you would like to work on in the Github tracker
   * Leave a comment letting us know you would like to work on it
 * Add tests:
-  * All too often code is developed to meet milestones which only undergoes
+  * All too often code developed to meet milestones only undergoes
   empirical, human testing
   * We would love to see unit tests throughout RITA
   * Currently we only have unit tests for Beacon check under analysis/beacon to
```

Comment:
We have tests in at least one other place. I'd maybe just change this to "There are a few sections of this project that currently have unit tests. [Here](https://github.com/activecm/rita/blob/master/analysis/beacon/analyzer_test.go) is a good example of how unit tests can be written

Reply:
Rewrote with the suggested wording and example unit tests.